packages/obconf.rb

```diff
@@ -0,0 +1,26 @@
+require 'package'
+
+class Obconf < Package
+  description 'ObConf - Openbox configuration tool'
+  homepage 'https://openbox.org/wiki/ObConf:About'
+  version '2.0.4'
+  compatibility 'all'
+  source_url 'https://openbox.org/dist/obconf/obconf-2.0.4.tar.gz'
+  source_sha256 '71a3e5f4ee246a27421ba85044f09d449f8de22680944ece9c471cd46a9356b9' 
+
+  depends_on 'gtk2'
+  depends_on 'openbox'
+
+  def self.build
+    system "./configure"
```

Comment:
Does `system "./configure #{CREW_OPTIONS}"` work?

Reply:
No need, as `obconf` sets `/usr/local/` as default install location, that's what exactly we need :) Also `#{CREW_OPTIONS}` causes a bunch of errors on building (but why?)
```shell
gcc: error: unrecognized command-line option ‘--libdir=/usr/local/lib64’
gcc: error: unrecognized command-line option ‘--mandir=/usr/local/share/man’
gcc: error: unrecognized command-line option ‘--libdir=/usr/local/lib64’
gcc: error: unrecognized command-line option ‘--libdir=/usr/local/lib64’
gcc: error: unrecognized command-line option ‘--libdir=/usr/local/lib64’
gcc: error: unrecognized command-line option ‘--build=x86_64-cros-linux-gnu’
gcc: error: unrecognized command-line option ‘--mandir=/usr/local/share/man’
gcc: error: unrecognized command-line option ‘--mandir=/usr/local/share/man’
gcc: error: unrecognized command-line option ‘--build=x86_64-cros-linux-gnu’
gcc: error: unrecognized command-line option ‘--mandir=/usr/local/share/man’
gcc: error: unrecognized command-line option ‘--host=x86_64-cros-linux-gnu’
gcc: error: unrecognized command-line option ‘--host=x86_64-cros-linux-gnu’
gcc: error: unrecognized command-line option ‘--target=x86_64-cros-linux-gnu/share/locale"’
gcc: error: unrecognized command-line option ‘--build=x86_64-cros-linux-gnu’
gcc: error: unrecognized command-line option ‘--target=x86_64-cros-linux-gnu/share/locale"’
gcc: error: unrecognized command-line option ‘--build=x86_64-cros-linux-gnu’
gcc: error: unrecognized command-line option ‘--libdir=/usr/local/lib64’
gcc: error: unrecognized command-line option ‘--libdir=/usr/local/lib64’
```